### PR TITLE
Clarify secrets in expressions and request header

### LINKED ISSUE #1057: Clarify secret handling for connectors
Update this page - https://docs.camunda.io/docs/components/console/manage-clusters/manage-secrets/

Secrets are not variables and therefore have to be surrounded by quotes. i.e. instead of = { myHeader: [secrets.MY](http://secrets.my/)_SECRET} it has to be = { myHeader: "[secrets.MY](http://secrets.my/)_SECRET"} which is surprising at first as people assume secrets are variables. 

Secrets are currently not supported in the body of a REST connector.

### PR DESCRIPTION
**Pull request by akeller**
Closes #1057

### CODE REVIEW

**Review by menski — APPROVED** [review]
:shipit:

**Comment by nikku** [comment]
Added a bunch of hints. Maybe you want to incorporate these.

TLDR: 

* Secrets are not variables but strings
* What this means, they must be wrapped in double quotes _inside feel expressions_.
